frontend/src/js/store/store.ts

```diff
@@ -121,7 +121,7 @@ export const getConfiguredStore = (options = {}) => {
         },
         serializableCheck: {
           ignoredActions: [organizationActions.receiveExternalDeviceIntegrations.name, setSnackbar.name, uploadProgress.name],
-          ignoredActionPaths: ['uploads', 'snackbar', /payload\..*$/],
+          ignoredActionPaths: ['uploads', 'snackbar', /payload\..*$/, 'meta.arg.file', 'meta.arg.integration.configHint'],
```

Comment:
We might also want to remove snackbar related actions from here, as we are not passing jsx there anymore